### PR TITLE
chore:  update notificationPortletVersion to 4.5.0

### PR DESCRIPTION
**Pull request by drewwills**
##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]

##### Description of change

Includes a new `notification-icon` web component based on Vue.

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/.github/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/.github/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

File: data/quickstart/portlet-definition/notification-icon.portlet-definition.xml [hunk 1]
```diff
@@ -80,8 +80,9 @@
         <readOnly>false</readOnly>
         <value>
             <![CDATA[
+                <script src="https://unpkg.com/vue@2.5.17/dist/vue.min.js"></script>
+                <script type="text/javascript" src="/NotificationPortlet/scripts/notification-icon.js"></script>
```
Comment (ChristianMurphy) [inline data/quickstart/portlet-definition/notification-icon.portlet-definition.xml]:
```suggestion
                <script type="text/javascript" src="/NotificationPortlet/scripts/notification-icon.min.js"></script>
```

File: data/quickstart/portlet-definition/notification-icon.portlet-definition.xml [hunk 2]
```diff
@@ -80,8 +80,9 @@
         <readOnly>false</readOnly>
         <value>
             <![CDATA[
+                <script src="https://unpkg.com/vue@2.5.17/dist/vue.min.js"></script>
```
Comment (ChristianMurphy) [inline data/quickstart/portlet-definition/notification-icon.portlet-definition.xml]:
```suggestion
                <script src="/resource-server/webjars/vue/dist/vue.min.js"></script>
```

Comment (drewwills) [reply data/quickstart/portlet-definition/notification-icon.portlet-definition.xml]: I think I mostly like this change -- when did we add Vue to the `resource-server`?

Comment (ChristianMurphy) [reply data/quickstart/portlet-definition/notification-icon.portlet-definition.xml]: https://github.com/Jasig/uPortal-start/pull/233

**Pull request merged**